docs/platform/configuration.md

```diff
@@ -0,0 +1,144 @@
+   specifies the actual configuration properties and their values.
+   You will need to consult the reference documentation of the target _type_ to known the list of available properties.
+
+3. The [**priority**]({{site.refDocsUrlPattern | replace: '$', 'pentaho.config.spec.IRule#priority'}}) 
+   allows fine-tuning the order by which rules that target the same _type_ are merged.
+   Higher values have higher priority. It is optional and defaults to `0`.
+
+See 
+[Rule Specificity]({{site.refDocsUrlPattern | replace: '$', 'pentaho.config.spec.IRuleSet' | append: '#Rule_Specificity'}}) 
+for more information on the the order by which configuration rules having the same target are merged.
+
+
+## Example Configuration Module
+
+The following is an AMD/RequireJS configuration module that contains two configuration rules:
+
+1. The first rule targets the _type_ `my.ICar`,
+when used by the `my-vehicle-editor` _application_,
+and specifies the value of its `tireSize` and `exteriorColor` properties.
+
+2. The second rule, has a higher-than-default _priority_, targets the _type_ `my.ICandy`,
+whatever the _application_ using it,
+and specifies the value of its `cocoaPercentage` and `fillingFlavour` properties.
+
+```js
+define(function() {
+  
+  "use strict";
+  
+  // The value of the module is an IRuleSet.
+  return {
+    rules: [
+      // IRule 1
+      {
+        select: {
+          type: "my.ICar",
+          application: "my-vehicle-editor"
+        },
+        apply: {
+          tireSize: 18,
+          exteriorColor: "caribbean-blue"
+        }
+      },
+    
+      // IRule 2
+      {
+        priority: 1,
+        select: {
+          type: "my.ICandy"
+        },
+        apply: {
+          cocoaPercentage: 0.9,
+          fillingFlavour: "orange"
+        }
+      }
+    ]
+  }
+});
+```
+
+
+## Global Configuration File
+
+_Ad hoc_ configuration rules can be added to the system, by system administrators, 
+by placing these in the **global configuration file** — 
+a configuration file conveniently registered for you.
+
+The file is located within the Apache Karaf folder at: `config/web-client/config.js`.
+Depending on the product, the Karaf folder is located at: 
+- Spoon PDI client: `data-integration/system/karaf/`.
+- Pentaho Server: `pentaho-server/pentaho-solutions/system/karaf/`.
+
+Editing and saving the file causes the system to refresh its configuration,
+without the need to restart the software.
+
+The configuration file is shipped with a small set of illustrative (but commented-out) rules.
+
+**ATTENTION**: **server upgrades overwrite this file** with an empty version of it, 
+so you need to backup the file yourself before upgrading and restore it afterwards.
+
+As an alternative to using the global configuration file, 
+you can bundle and deploy your own [Pentaho Web Package](pentaho-web-package) 
+containing a registered configuration module.
+
```

Comment:
I know this doesn't say otherwise but I would like to make it explicit that the configuration _can_ be "bundled" together with other stuff, like a visualisation.